src/internal_events/log_to_metric.rs

```diff
@@ -59,3 +59,84 @@ impl<'a> InternalEvent for LogToMetricParseFloatError<'a> {
         emit!(ComponentEventsDropped::<UNINTENTIONAL> { count: 1, reason })
     }
 }
+
+//  Metric Metadata Events and Errors
+pub struct MetricMetadataInvalidFieldValueError<'a> {
+    pub field: &'a str,
+    pub field_value: &'a str,
+}
+
+impl<'a> InternalEvent for MetricMetadataInvalidFieldValueError<'a> {
```

Comment:
Perhaps someone with more expertise on Vector metrics wants to review the changes to this file (cc @jszwedko)